### ISSUE TITLE
Make an update script using get_filings.R

### ISSUE THREAD

**Issue by iangow** [iangow-public/edgar#8]
#Take a look at [`get_filings.R`](https://github.com/iangow-public/edgar/blob/dabb893e0f9c7582be1c7dcf69f87cf00e578a8b/get_filings.R).

The code is broken into three "chunks". The first chunk creates functions. The second chuck updates all years from 1994-2017 and the third chunk updates the first quarter of 2018.

A better approach might be to:

1. Determine the current quarter.
2. Determine the most recent quarter on the database (`edgar.filings`).
3. Update every quarter from the last quarter on the database to the current quarter (inclusive).

It might be worth examining what happens when you run the following after running the first chunk of code (don't run the second chunk!).

```
temp <- getSECIndexFile(year = 2018L, quarter = 1L)
```

Here is some code for step 2:

``` r
library(dplyr, warn.conflicts = FALSE)
library(RPostgreSQL)
#> Loading required package: DBI
pg <- dbConnect(PostgreSQL())

filings <- tbl(pg, sql("SELECT * FROM edgar.filings"))

last_quarter <-
    filings %>% 
    mutate(year = date_part('year', date_filed), 
           quarter = date_part('quarter', date_filed)) %>%
    filter(year == max(year, na.rm = TRUE)) %>%
    summarise(year = max(year, na.rm = TRUE), 
              quarter = max(quarter, na.rm = TRUE)) %>%
    collect()

last_quarter
#> # A tibble: 1 x 2
#>    year quarter
#> * <dbl>   <dbl>
#> 1 2018.      1.
last_quarter$year[1]
#> [1] 2018
last_quarter$quarter[1]
#> [1] 1
```

Created on 2018-03-15 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0).

**Comment by iangow** [comment]
And for step 1 above, this would be one approach.

``` r
library(lubridate)
#> 
#> Attaching package: 'lubridate'
#> The following object is masked from 'package:base':
#> 
#>     date
quarter(today())
#> [1] 1
year(today())
#> [1] 2018
```

Created on 2018-03-15 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0).

**Comment by iangow** [comment]
Oops. I had linked to the wrong file above. I've now fixed this.

**Comment by iangow** [comment]
This issue was addressed as part of #9.
